src/utilities/download_TROPOMI.py

```diff
@@ -15,186 +18,62 @@ def download_TROPOMI(startdate, enddate, Sat_datadir):
         Sat_datadir  [str]            : TROPOMI data directory for storing data
 
     """
-    # offline: 11/28/18 to end of 2020? + parts of 01/21
-    # s3://meeo-s5p/OFFL/L2__CH4___/2018/MM/DD/*.nc
-    # reprocessed: 04/30/18 to 11/28/18
-    # s3://meeo-s5p/RPRO/L2__CH4___/2018/MM/DD/*.nc
+    # offline: 07/26/2022 to present
+    # s3://meeo-s5p/OFFL/L2__CH4___/YYYY/MM/DD/*.nc
+    # reprocessed: 04/30/2018 to 07/25/2022
+    # s3://meeo-s5p/RPRO/L2__CH4___/YYYY/MM/DD/*.nc
     # --no-sign-request
-    start_str = str(startdate)
-    start_year = start_str[:4]
-    start_month = start_str[5:7]
-    start_day = start_str[8:10]
 
-    end_str = str(enddate)
-    end_year = end_str[:4]
-    end_month = end_str[5:7]
-    end_day = end_str[8:10]
+    # list of all dates between startdate and enddate excluding enddate
+    download_dates = np.arange(startdate, enddate, dtype="datetime64[D]")
```

Comment:
I think you might want `download_dates = np.arange(startdate, enddate + np.timedelta64(1, 'D'), dtype="datetime64[D]")`.

Reply:
I think you might be right. I was mimicking the previous script, which excludes the end date, but I think we would want this to be inclusive. @djvaron thoughts?

Reply:
I'm not so sure. Currently our inversions end at hour 0 of the end date. So if I want a 1-week inversion starting on 2018-05-01, I set `startdate = 20180501` and `enddate = 20180508`. Days 1-7 are complete, and day 8 includes just hour 0. This is probably why we exclude the `enddate` -- right?